xdcs-frontend/src/app/view/task-creation/task-creation.component.html

```diff
@@ -0,0 +1,155 @@
+<app-navbar></app-navbar>
+<ng-container *ngIf="isDefinitionConfigured(); else notSelected">
```

Comment:
1. wywołanie metody w tym miejscu to raczej zły pomysł
2. jeśli to zwróci false to co user powinien zrobić?
3. podczas startowania taska startujemy go z deploymentu a nie definicji

Reply:
4. dla deploymentu to już nie jest możliwe żeby nie była ustawiona konfiguracja bo to powinno być walidowane podczas deployu

Reply:
dalej może mieć puste pola kernelname i kernelfile

Reply:
1. bo angular nie będzie w stanie się zbindować do wartości i będzie musiał za każdym razem niepotrzebnie wykonywać tą funkcję
2. ale to dodamy walidację przy deployu na te pola

Reply:
done